src/Microsoft.AspNetCore.Mvc.Localization/DependencyInjection/MvcLocalizationMvcBuilderExtensions.cs

```diff
@@ -86,5 +87,66 @@ public static IMvcBuilder AddViewLocalization(this IMvcBuilder builder)
             MvcLocalizationServices.AddLocalizationServices(builder.Services, format, setupAction);
             return builder;
         }
+
+        /// <summary>
+        /// Adds MVC view and data annotations localization services to the application.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcBuilder"/>.</param>
+        /// <returns>The <see cref="IMvcBuilder"/>.</returns>
+        public static IMvcBuilder AddMvcLocalization(this IMvcBuilder builder)
+        {
+            if (builder == null)
+            {
+                throw new ArgumentNullException(nameof(builder));
+            }
+
+            return AddMvcLocalization(builder, localizationOptionsSetupAction: null);
+        }
+
+        /// <summary>
+        /// Adds MVC view and data annotations localization services to the application.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcBuilder"/>.</param>
+        /// <param name="localizationOptionsSetupAction">An action to configure the <see cref="LocalizationOptions"/>.</param>
+        /// <returns>The <see cref="IMvcBuilder"/>.</returns>
+        public static IMvcBuilder AddMvcLocalization(
+            this IMvcBuilder builder,
+            Action<LocalizationOptions> localizationOptionsSetupAction)
+        {
+            if (builder == null)
+            {
+                throw new ArgumentNullException(nameof(builder));
+            }
+
+            return AddMvcLocalization(
+                builder,
+                localizationOptionsSetupAction,
+                LanguageViewLocationExpanderFormat.Suffix,
+                dataAnnotationsLocalizationOptionsSetupAction: null);
+        }
+
+        /// <summary>
+        /// Adds MVC view and data annotations localization services to the application.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcBuilder"/>.</param>
+        /// <param name="format">The view format for localized views.</param>
+        /// <param name="localizationOptionsSetupAction">An action to configure the <see cref="LocalizationOptions"/>.</param>
+        /// <param name="dataAnnotationsLocalizationOptionsSetupAction">An action to configure the <see cref="MvcDataAnnotationsLocalizationOptions"/>.</param>
+        /// <returns>The <see cref="IMvcBuilder"/>.</returns>
+        public static IMvcBuilder AddMvcLocalization(
+            this IMvcBuilder builder,
+            Action<LocalizationOptions> localizationOptionsSetupAction,
+            LanguageViewLocationExpanderFormat format,
+            Action<MvcDataAnnotationsLocalizationOptions> dataAnnotationsLocalizationOptionsSetupAction)
```

Comment:
Don't add overloads for all combinations of these parameters. But, please add `AddMvcLocalization(IMvcBuilder, LanguageViewLocationExpanderFormat)` and `AddMvcLocalization(IMvcBuilder, Action<MvcDataAnnotationsLocalizationOptions>)`.

Reply:
I didn't want to add these overloads because I didn't feel they are commonly used. Felt the one overload accepting the `localizationOptions` is common as users typically set path to where their `Resources` are.

Reply:
What are you basing "typically" on? `AddViewLocalization(...)` has analogous overloads.

Reply:
I'm ok with any of:
1. Add overloads for every combo
2. Add all overloads except those that have a reasonable justification to not include

I don't feel strongly about which of those two we go with.

Reply:
Let's finish off with option 2; it's 3 more methods in each extensions class. I've crossed out the first sentence of my initial comment here.